### ISSUE TITLE
SpinViewer with latest Vue3 does not work in production builds

### ISSUE THREAD

**Issue by kbs1** [naver/egjs-view360#397]
## Description
When trying to use SpinViewer in latest Vue3 production build, it is unable to get it's props. In debug build, the following message appears in console:
```
[Vue warn]: Avoid app logic that relies on enumerating keys on a component instance. The keys will be empty in production mode to avoid performance overhead.
  | PublicInstanceProxyHandlers.ownKeys | @ | runtime-core.esm-bundler.js?d2dd:3252
  | getValidProps | @ | view360.esm.js?562e:7346
  | mounted | @ | view360.esm.js?ffb2:144
```
This happens in `getValidProps` method.

## Steps to check or reproduce
Try to use in production build with Vue3.

**Comment by kbs1** [comment]
Bug seems to be here: https://github.com/naver/egjs-view360/blob/master/packages/vue3-view360/src/SpinViewer.ts#L40 - should be `const props = getValidProps(this.$props);`

**Comment by WoodNeck** [comment]
Thanks for reporting the issue :)
I've made a PR(#398) that fixes this issue.

**Comment by kbs1** [comment]
Hi, thank you a lot! :) Do you think this simple fix could be merged soon?

**Comment by WoodNeck** [comment]
Sure, as soon as the review is finished

**Comment by WoodNeck** [comment]
Just released a new version (v3.0.4) that fixes this issue, try that one!
